Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:20.15.1
+
+WORKDIR /app
+
+# Install pnpm (matches packageManager in package.json)
+RUN npm install -g pnpm@9.4.0
+
+# Copy package files first
+COPY package.json pnpm-lock.yaml ./
+
+# Clean install dependencies with legacy peer deps
+RUN rm -rf node_modules
+RUN pnpm install --no-frozen-lockfile
+RUN pnpm rebuild
```

Comment:
Should this be a multistage build?

Reply:
It doesn't have to be multistage. Realistically 99% of the people who use Bolt.new locally will be for personal use to build their own apps not for a live production version of Bolt therefore to me it didn't make sense to separate this out into a development and production multistage.